api/controllers/EAppEligibilityQuestionsController.js

```diff
@@ -0,0 +1,100 @@
+const EAppEligibilityQuestionsController = {
+    renderEligibilityQuestion(req, res) {
+        const eligibilityViews = {
+            'apostille-accepted-in-desitnation':
+                'eApostilles/eligibilityQuestionOne.ejs',
+            'documents-eligible-for-service':
+                'eApostilles/eligibilityQuestionTwo.ejs',
+            'pdfs-digitally-signed':
+                'eApostilles/eligibilityQuestionThree.ejs',
+        };
+
+        const questionPage = eligibilityViews[req.param('question')];
```

Comment:
if a user manually inputs into their url bar "....gov.uk/eligibility/something-that-doesnt-exist" - will it show a user a 404? I'm suspecting that it might end up giving the user a 500 (internal server error) which wouldn't be correct. Maybe default to the first page?

Reply:
Yeah looks like it's taken care of by the Sailsjs 404 page. Not the best 404 message but it's something 🤷 

<img width="1080" alt="Screenshot 2021-09-02 at 18 07 22" src="https://user-images.githubusercontent.com/1377253/131887173-7ecd2dc0-a36d-4904-a453-56d9e2c44fdb.png">

Actually, looks like that's the 403 page. This project is so random 🤦

Reply:
Okay, should be fixed to go to 404 now.

<img width="1025" alt="Screenshot 2021-09-02 at 18 33 46" src="https://user-images.githubusercontent.com/1377253/131890486-8130f0be-042f-4362-9c82-dc391cfbb51d.png">